python/lsst/sims/featureScheduler/comcamTessellate.py

```diff
@@ -0,0 +1,51 @@
+import numpy as np
+
+
+def comcamTessellate(side_length=0.7, overlap=0.11):
```

Comment:
I figured someone would come up with better tessellation schemes and they could all go in here. If there's one obvious best way, then it would be good to move to `utils`.